compiler/gentree.h

```diff
@@ -152,4 +154,3 @@ class GenTree {
   FunctionPtr cur_function;         // = functions_stack.back()
   SrcFilePtr processing_file;
 };
-
```

Comment:
Unnecessary change

Reply:
This file has two empty line in the end file - it's bad. Therefore I delete extra empty line.